### PR TITLE
W-11673772 upcomingprocesschanges se

### PR DESCRIPTION
**Pull request by sebermuley**
- Added new page **Upcoming Process Changes** to topic **Process Operations** in the RPA Manager docs
-- Topics:
--- **View Upcoming Process Changes**
--- **Map BPMN Elements to Process Activities** (with example image)
--- **Deploy the Latest Version of an Activity**
--- **Deploy A Prior Version of an Activity**
- Added new topic **Change the Revision of a Referenced Activity** to the page **Using the Activity Library** in the RPA Builder docs

### CODE REVIEW

**Comment by sebermuley** [comment]
SME reviewed by Frederik Laurentzsch

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
+= Upcoming Process Changes
+
+In the activity library of the RPA Builder, RPA developers save activities that can be reused in different processes, such as login procedures or creations of data sets. To reuse a saved activity, an RPA developer can import a reference to this activity into a bot task.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:3]:
```suggestion
The activity library of RPA Builder enables RPA developers to save activities that can be reused in different processes, such as login procedures or the creation of data sets. To reuse a saved activity, an RPA developer can import a reference to this activity into a bot task. 
```

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc [hunk 2]
```diff
@@ -53,6 +53,17 @@ You cannot import an activity that contains activity parameters that use the sam
 +
 This method creates a local copy of the workflow behind the activity and inserts it into the project, which enables you to make changes. However, because there is no reference to the activity in the Activity Library, the local copy does not automatically update if the activity in the Activity Library changes. 
 
+[[change-the-revision-of-a-referenced-activity]]
+== Change the Revision of a Referenced Activity
+
+If the latest revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], you can revert the activity to another revision:
```
Comment (Cristian-Venticinque) [inline rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:59]:
```suggestion
If the latest revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], follow these steps to revert the activity to an earlier revision:
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
+= Upcoming Process Changes
+
+In the activity library of the RPA Builder, RPA developers save activities that can be reused in different processes, such as login procedures or creations of data sets. To reuse a saved activity, an RPA developer can import a reference to this activity into a bot task. 
+
+If an RPA developer edits a referenced activity, RPA Manager shows a label with the number of changed activities next to the navigation items *Process Operations* and *Upcoming Process Changes*. 
+
+In the view *Upcoming Process Changes* you can see which processes contain the changed activities and deploy the latest version.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:7]:
```suggestion
The *Upcoming Process Changes* shows which processes contain the changed activities and enables you to deploy the latest version. 
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+[[view-upcoming-process-changes]]
+== View Upcoming Process Changes 
+
+If you see a label with a number next to the *Process Operations* or *Upcoming Process Changes* navigation entry, an RPA developer has edited a deployed activity in the activity library in RPA Builder.
+
+To view the changes:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. View the descriptions in the table.
+. To compare the deployed version with the latest version of an activity, click the *Show Changes* icon image:show-icon.png[eye symbol,1.5%,1.5%] in the table row of the edited activity.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:36]:
```suggestion
. Click the *Show Changes* icon image:show-icon.png[eye symbol,1.5%,1.5%] in the table row of the edited activity to compare the deployed version with the latest version of that activity.
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+[[map-bpmn-elements-to-process-activities]]
+== Map BPMN Elements to Process Activities
+
+If a process contains the same activity more than once, use your browser's developer tools to find out which BPMN element contains the referenced activity revision:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. Right-click on a process name, open the project in another tab and open the *Automation* panel there.
+. In the *Automation* panel, right-click on one of the possible activities and choose *Inspect* from the browser's context menu.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:45]:
```suggestion
. Right-click on a process name and open the project in another tab.
. In the new tab, open the *Automation* panel.
. Right-click on one of the possible activities and choose *Inspect* from the browser's context menu. 
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+. Compare the *data-element-id* with the *BPMN Element* id in the list of all process changes.
+
+Matching activities have the same id.
+ 
+image::rpa_processOperations_UpcomingProcessChanges_WhichBPMNElement.png[Screenshots of the Upcoming Process Changes table and the Automation Panel's developer tools elements view with an element highlighted. An arrow shows matching IDs.]
+
+[[deploy-the-latest-version-of-an-activity]]
+== Deploy the Latest Version of an Activity
+
+To ensure that the processes use an appropriate version of a referenced activity, you can deploy the latest version to processes in the design, build, or test phase.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:62]:
```suggestion
To ensure that the processes use an appropriate version of a referenced activity, deploy the latest version to processes in the design, build, or test phase.
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+To deploy the latest version of an activity:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. Select the activities to deploy.
+. Click *Deploy*.
+
+Test the updated processes before transfering them to production phase.
+
+== Deploy A Prior Version of an Activity
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:78]:
```suggestion
== Deploy an Earlier Version of an Activity
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+To ensure that a process uses an appropriate version of a referenced activity or to undo a prior deployment, you can change the version of a referenced activity in the RPA Builder:
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:80]:
```suggestion
To ensure that a process uses an appropriate version of a referenced activity or to undo a prior deployment, change the version of a referenced activity in RPA Builder:
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+. Transfer the process to the build phase.
+. Open the process in the RPA Builder.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:83]:
```suggestion
. Open the process in RPA Builder.
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+. xref:rpa-builder::activity-library.adoc#change-the-revision-of-a-referenced-activity[Change the revision of the referenced activity using RPA Builder.]
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:84]:
```suggestion
. xref:rpa-builder::activity-library.adoc#change-the-revision-of-a-referenced-activity[Change the revision of the referenced activity using RPA Builder].
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+. Upload the process to the RPA Manager.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:85]:
```suggestion
. Upload the process to RPA Manager.
```

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+. Transfer the process to the next phase. 
+
+Test the updated process before transfering it to production phase.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:88]:
```suggestion
Test the updated process before transferring it to the production phase.
```

**Review by Cristian-Venticinque — APPROVED** [review]
Please check suggestions before merging.

**Review by Cristian-Venticinque — DISMISSED** [review]
Please check suggestions before merging.

File: rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc (hunk 2 above)
Comment (sebermuley) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:59]: You can also revert the activity to a later revision again.

Comment (sebermuley) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:59]: --> replaced with: to an earlier or later revision

Comment (sebermuley) [reply rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc:59]: If the deployed revision of a referenced activity does not work properly in a process or if you accidentally xref:rpa-manager::processoperations-upcomingprocesschanges.adoc#deploy-the-latest-version-of-an-activity[deployed the latest revision using RPA Manager], follow these steps to revert the activity to an earlier or later revision:

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+|Upcoming Process Changes Administration
+
+|===
+
+* To deploy a process change, the project must be in the design, build, or test phase.
+
+[[view-upcoming-process-changes]]
+== View Upcoming Process Changes 
+
+If you see a label with a number next to the *Process Operations* or *Upcoming Process Changes* navigation entry, an RPA developer has edited a deployed activity in the activity library in RPA Builder.
```
[the remark this answers is not in the document]
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:30]: added: with the number of affected processes

File: rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc (hunk 9 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc:83]: Changed every occurence of
the RPA Manager/Builder/Recorder in the RPA Manager docs
(and implemented som vale suggestions)

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**